content/en/docs/concepts/extend-kubernetes/compute-storage-net/resource-drivers.md

```diff
@@ -0,0 +1,278 @@
+---
+title: DRA Resource Drivers
+description: Resource drivers provide non-trivial allocation logic and management for devices or resources that require vendor-specific or just complex setup, such as GPUs, NICs, FPGAs, etc.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+Kubernetes provides a
+[Dynamic Resource Allocation (DRA)](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/3063-dynamic-resource-allocation)
+that can be leveraged to provide more complex hardware resources to workloads with custom resource
+accounting.
+
+Similarly to {{< glossary_tooltip term_id="device-plugin" text="Device Plugins">}}, instead of
+customizing the code for Kubernetes itself, vendors can implement a resource driver that you deploy
+into the cluster to account for and control the allocation of GPUs, high-performance NICs, FPGAs,
+InfiniBand adapters, and other similar computing resources that may require vendor specific
+initialization and setup.
+
+With device plugins, the scheduler was given a trivial, numerical representation of the resources
+available on node for consideration during scheduling as an extended resource.
+
+With DRA, the scheduler is offloading the task of allocating and accounting for non-native resources
+to the resource driver, which manages such resources in the cluster.
+
+Resource driver consists of two main components:
+
+- controller (one per cluster), manages hardware resources allocation for
+  {{< glossary_tooltip term_id="resource-claim" text="Resource Claims">}}
+- kubelet-plugin (one per node with resource), that:
+  - discovers the supported hardware
+  - announces discovered hardware to the Resource Driver controller
+  - prepares the hardware allocated for Pod when Pod comes to the Kubelet
+  - unprepares the hardware allocated for Pod when Pod has reached final state or is being deleted.
+
+Communications between controller and kubelet-plugin can be done in two ways:
+
+- through Custom Resource Definition objects provided by and designed for the resource driver
+- through ResourceHandle which is a part of AllocationResult provided by the controller in case of
+  successful allocation
+
+General recommendations:
```

Comment:
Should this be a heading?

Reply:
Perhaps if / when we get more bullets in it, now seems unnecessary.